archimedes/archimedes/functions/list/manipulations.py

```diff
@@ -1,16 +1,20 @@
 from collections.abc import Iterable
+import re 
 
 def unique(ls):
     return list(dict.fromkeys(ls))
 
-# def flatten(ls):
-#     return list(itertools.chain.from_iterable(ls))
-
 def flatten(ls):
     def _flatten(l):
         for el in l:
             if isinstance(el, Iterable) and not isinstance(el, (str, bytes)):
                 yield from _flatten(el)
             else:
                 yield el
-    return list(_flatten(ls))
\ No newline at end of file
+    return list(_flatten(ls))
+
+def order(ls): 
```

Comment:
I'd document that ls here is intended ot be a list of strings -- any other list will bail on the convert line here.

Reply:
looks like you commented somehow on the outdated version, but point taken and I'll put that where it's needed now.